rancher/rancher-services/storage-service/index.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Storage Service
+layout: rancher-default
+
+---
+
+## Storage Service
+
+_Available as of v0.47.0+_
+
+As part of the [Rancher catalog]({{site.baseurl}}/rancher/rancher-ui/applications/catalog/), Rancher provides a storage service. Currently, the storage services are pre-fixed with `Convoy` in the catalog. 
+
+### Prerequisites
+* You have the underlying storage system properly deployed and accessible to Rancher services
+
+### Limitations
+
+* Only one Convoy service can be launched in one Rancher [environment]({{site.baseurl}}/rancher/configuration/environments/). 
+
+### Setting up the Storage Service
+
+#### Adding Host Labels
+
+Prior to launching storage services, it's recommended to prepare the hosts that will be able to use shared Docker volumes. For every host, add a [host label]({{site.baseurl}}/rancher/rancher-ui/infrastructure/hosts/#host-labels). After the labels are added to all hosts, you are ready to launch the storage service.
```

Comment:
Before launching storage services,  you must prepare the hosts that can use shared Docker volumes. For every host, add a host label.  After labels are added to all the hosts, you are ready to launch the storage service.